Makefile

```diff
@@ -9,7 +9,15 @@ include $(SHIPYARD_DIR)/Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e / -e reload-images)
 override BUILD_ARGS += $(shell source ${SCRIPTS_DIR}/lib/version; echo --ldflags \'-X main.VERSION=$${VERSION}\')
-override CLUSTERS_ARGS += --cluster_settings $(DAPPER_SOURCE)/scripts/cluster_settings
+
+ifneq (,$(filter ovn,$(_using)))
+override CLUSTER_SETTINGS_FLAG = --cluster_settings $(DAPPER_SOURCE)/scripts/cluster_settings.ovn
+else
+override CLUSTER_SETTINGS_FLAG = --cluster_settings $(DAPPER_SOURCE)/scripts/cluster_settings
+endif
+
+override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
+override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
```

Comment:
Isn't this handled by your Shipyard PR? https://github.com/submariner-io/shipyard/pull/398/files

Reply:
> Isn't this handled by your Shipyard PR? [submariner-io/shipyard#398](https://github.com/submariner-io/shipyard/pull/398)

This change is in `shipyard/Makefile`, while we include `shipyard/Makefile.inc` Without this change cluster_settings were always empty in `deploy.sh` and it wasn't picking up our changes.

Reply:
Ah, I see. So should we move that from `shipyard/Makefile` to `shipyard/Makefile.inc` so it's available here (and eventually in the Operator I imagine)? That's where we have similar `using` flag logic.

Reply:
In the shipyard patch Mike said not to keep it in Makefile.inc IIRC, it is coz we're including a file that is local to repo, just like Makefile. Makefile.inc is for code and files that are shared by all. Other reason was that we probably won't be using it in lighthouse repo.

Reply:
Humm, okay. I'll try out a PR in Shipyard with my proposal and we can discuss there vs blocking you here.

Reply:
> Humm, okay. I'll try out a PR in Shipyard with my proposal and we can discuss there vs blocking you here.

Following up, WIP proposal here: https://github.com/submariner-io/shipyard/pull/441